aries-site/src/examples/components/layer/LayerSideDrawerExample.js

```diff
@@ -131,8 +130,6 @@ export const LayerSideDrawerExample = () => {
           position="right"
           full={size !== 'small' ? 'vertical' : true}
           modal={false}
```

Comment:
The Figma file shows this as being a modal.

On a tangent, if we make this a modal as well, then all examples are modal layers. Do we have use cases where the layer should not be a modal? Should we be showing them? Do we have guidance around when the page content should remain active vs. when a modal should be used to disable interaction?

Reply:
It looks like we have the guidance above the example, but the example is not following the guidance completely.